test/integration/kubeadm/token_test.go

```diff
@@ -22,20 +22,29 @@ import (
 	"path/filepath"
 	"regexp"
 	"testing"
+
+	_ "github.com/golang/glog"
```

Comment:
its required because of the flags that `make test-integration` pass to it. 

https://github.com/kubernetes/kubernetes/blob/master/hack/make-rules/test-integration.sh#L66